src/overmap.cpp

```diff
@@ -919,6 +919,14 @@ struct enum_traits<cube_direction> {
     static constexpr cube_direction last = cube_direction::last;
 };
 
+struct cube_direction_hash {
+    std::size_t operator()( cube_direction d ) const {
+        return static_cast<std::size_t>( d );
+    }
+};
```

Comment:
Suggest specializing `std::hash` rather than creating a separate one, so that we don't need to change the code that uses it as a key.

Reply:
done